src/test/kotlin/uk/gov/justice/digital/hmpps/crimeportalgateway/endpoint/ExternalDocRequestEndpointTest.kt

```diff
@@ -69,10 +70,26 @@ internal class ExternalDocRequestEndpointTest {
         verifyNoMoreInteractions(sqsService, telemetryService)
     }
 
+    @Test
+    fun `given a valid message with dummy court room then should enqueue the message and return the correct acknowledgement`() {
+
+        endpoint = buildEndpoint(setOf("B10JQ"), false, 5)
+        whenever(sqsService.enqueueMessage(contains("ExternalDocumentRequest")))
+            .thenReturn("a4e9ab53-f8aa-bf2c-7291-d0293a8b0d02")
+
+        val ack = endpoint.processRequest(externalDocument)
+
+        assertAck(ack)
+
+        verify(telemetryService).trackEvent(TelemetryEventType.COURT_LIST_MESSAGE_RECEIVED, customDimensionsMap)
+        verify(sqsService).enqueueMessage(anyString())
```

Comment:
See previous comment, aren't we expecting these to be filtered out?

Reply:
I remove that and the test still passes. 😳 Yes - you are right.